app/javascript/gobierto_data/webapp/components/commons/Info.vue

```diff
@@ -141,9 +139,8 @@ export default {
   },
   computed: {
     compiledHTMLMarkdown() {
-      const turndownService = new TurndownService()
-      const markdown = turndownService.turndown(this.descriptionDataset)
-      const mdText = marked(markdown, {
+      const descriptionHTML = this.descriptionDataset.replace(/\|<br>/g, '|').replace(/<p>\|/g, '|').replace(/\|<\/p>/g, '|');
```

Comment:
Since you're using regex as well as the substitution string is always the same, you can do it all at once for perfomance:
```suggestion
      const descriptionHTML = this.descriptionDataset.replace(/\|<br>|<p>\||\|<\/p>/g, '|');
```
Also, an explanatory comment would be nice

Reply:
Done!